test/decl/func/functions.swift

```diff
@@ -95,6 +95,16 @@ func parenPatternInArg((a): Int) -> Int { // expected-error {{expected parameter
 }
 parenPatternInArg(0)  // expected-error {{argument passed to call that takes no arguments}}
 
+func nullaryFunctionFixit() {}
+nullaryFunctionFixit(())
```

Comment:
Here, however, the same error has to be raised, but it doesn't happen! When I run the compiler, it does raise an error. When I run the test, it passes without expecting any error.